content/chainguard/chainguard-images/overview.md

```diff
@@ -27,7 +27,7 @@ Main features include:
 - Verifiable signatures provided by [Sigstore](/open-source/sigstore/cosign/an-introduction-to-cosign/)
 - Automated nightly builds to ensure images are completely up-to-date and contain all available security patches
 
-Chainguard Images are available [on Github](https://github.com/chainguard-images) and the `cgr.dev` registry.
+Chainguard Images are available from the [Chainguard Registry](/chainguard/chainguard-images/registry/overview/) and [on GitHub](https://github.com/chainguard-images).
```

Comment:
The config is available on GitHub but the images should be pulled from CGR. I'm not sure the best way to phrase that, any ideas?

Reply:
I updated to read this, but we can also direct people to the automated docs if you prefer?

```
Chainguard Images are available from the [Chainguard Registry](/chainguard/chainguard-images/registry/overview/) and can be pulled from `cgr.dev`. You can review images files [on GitHub](https://github.com/chainguard-images).
```